### PR TITLE
feat: my collection more local filters

### PR DESCRIPTION
**Pull request by brainbicycle**
The type of this PR is: **feat**

This PR resolves [CX-2136]

### Description

Adds a couple more local filters and one more sort:
- price paid sort
- price paid filter
- size filter
- rarity filter

Not included:
- location filter, this still needs to be added to the form, uses artworkLocation field
- materials filter, also needs to be added to the form
- time period filter, major_period needs to be exposed on backend
- demand_index sort, this is not part of artwork part of market_stats so might be tricky
- recently_updated

Ticket for rest of filters:
https://artsyproduct.atlassian.net/browse/CX-2234

<details><summary>Screenshots</summary>

local-sort-all-filters
local-sort-sorts
local-sort-rarity
local-sort-price
local-sort-size

</details>

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Add more local sorts + filters to my collection - Brian

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-2136]: https://artsyproduct.atlassian.net/browse/CX-2136?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Add more local sorts + filters to my collection - Brian)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 05f511d4b136ff699a98417903c07a6d552a5979
</p>

File: Artsy.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3233,6 +3233,8 @@
 				"${PODS_ROOT}/Artsy+UIFonts/Pod/Assets/Unica77LL-Regular.otf",
 				"${PODS_ROOT}/Artsy+UILabels/Pod/Assets/Chevron_Black.png",
 				"${PODS_ROOT}/Artsy+UILabels/Pod/Assets/Chevron_Black@2x.png",
+				"${PODS_ROOT}/../emission/Pod/Assets/Emission.js",
```
Comment (pvinis) [inline Artsy.xcodeproj/project.pbxproj:3236]: should we revert this file? this hasnt popped up in a while 🤔

File: Artsy.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -3264,6 +3266,8 @@
 				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Unica77LL-Regular.otf",
 				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Chevron_Black.png",
 				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Chevron_Black@2x.png",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/Emission.js",
+				"${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/assets",
```
Comment (pvinis) [inline Artsy.xcodeproj/project.pbxproj:3270]:
```suggestion
```

File: Artsy.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -3233,6 +3233,8 @@
 				"${PODS_ROOT}/Artsy+UIFonts/Pod/Assets/Unica77LL-Regular.otf",
 				"${PODS_ROOT}/Artsy+UILabels/Pod/Assets/Chevron_Black.png",
 				"${PODS_ROOT}/Artsy+UILabels/Pod/Assets/Chevron_Black@2x.png",
+				"${PODS_ROOT}/../emission/Pod/Assets/Emission.js",
+				"${PODS_ROOT}/../emission/Pod/Assets/assets",
```
Comment (pvinis) [inline Artsy.xcodeproj/project.pbxproj:3237]:
```suggestion
```

**Review by pvinis — APPROVED** [review]
I reverted that file and ill merge

**Comment by pvinis** [comment]
#squashongreen

**Comment by brainbicycle** [comment]
> I reverted that file and ill merge

thanks!!!

**Pull request merged**